packages/main/src/themes/TabContainer.css

```diff
@@ -126,10 +126,4 @@
 
 :host([media-range="XL"]) .ui5-tc__content {
 	padding: 1rem 3rem;
-}
-
-/*** RTL ***/
-[dir=rtl] .ui5-tc__overflow {
-	margin-right: auto;
-	margin-left: 0.25rem;
-}
+}
```

Comment:
In `/packages/main/src/themes/TabInStrip.css`, could you also change the following lines:

```css
.ui5-tab-expand-button > [ui5-button] {
	height: 1.5rem;
	min-width: 1.5rem;
	margin-top: 0.25rem;
-       margin-left: 0.625rem;
+       margin-inline-start: 0.625rem;
 }

       . . .

 .ui5-tab-single-click-icon {
-       margin-left: 0.875rem;
+       margin-inline-start: 0.875rem;
 }
```